napari/layers/labels/_tests/test_labels.py

```diff
@@ -258,6 +258,17 @@ def test_properties():
     assert layer_message.endswith('Class 12')
 
 
+def test_default_properties_assignment():
+    """Test that the default properties value can be assigned to properties
+    see https://github.com/napari/napari/issues/2477
+    """
+    np.random.seed(0)
+    data = np.random.randint(20, size=(10, 15))
+
+    layer = Labels(data)
+    layer.properties = layer.properties
```

Comment:
Is the default `layer.properties` an empty dict `{}`?

The test might be a little bit more readable if it was

```suggestion
    layer.properties = {}
    assert layer.properties == {}
```